### PR TITLE
includes schemaErrors as a field in clinical entity

### PR DESCRIPTION
**Pull request by hlminh2000**
includes schemaErrors as a field in clinical entity

### CODE REVIEW

File: schemas/Clinical/index.js [hunk 1]
```diff
@@ -257,20 +258,15 @@ const convertClinicalSubmissionDataToGql = data => {
...
-  }
-  // collect schema errors for each entity in dataErrors (not sure if this is OK??)
-  for (let clinicalType in schemaErrors) {
-    clinicalEntities.push(
-      convertClinicalSubmissionEntityToGql(clinicalType, {
-        dataErrors: schemaErrors[clinicalType],
-      }),
-    );
-  }
+  const clinicalEntities = Object.entries(submission.clinicalEntities).map(
```
Comment (jaserud) [inline schemas/Clinical/index.js]: Hmm `Object.entries(submission.clinicalEntities)` can throw error. If submission doesn't exist in DB then default value is used from `const submission = get(data, 'submission', {});` where `submission.clinicalEntities` doesn't exist

Comment (hlminh2000) [reply schemas/Clinical/index.js]: 👍 nice catch, thanks!

Comment (hlminh2000) [reply schemas/Clinical/index.js]: @jaserud address! :)

Comment (jaserud) [reply schemas/Clinical/index.js]: :+1:

**Pull request merged**